esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1468,9 +1468,24 @@ void Esdl2Transformer::addMethod(Esdl2Base *item)
     }
 }
 
+void Esdl2Transformer::setMethod(const char *name, Esdl2Method *method)
```

Comment:
Name is inconsistent with readMethodInfo and queryMethodInfo. Also inconsistent with addType relative to readType and queryType. Should it be addMethodInfo?

Reply:
I'll change to setMethodInfo() set because it is setting the value saved with name to be equal to this methodInfo.  addType just takes a type and doesn't directly set the name value pair.. although it does it internally by calling a set method.

Reply:
set in this case means a one to one relationship.  name=value.  In my general way of thinking, "append" or "add" would be a one to many or many to many relationship.